task/09-functions-n-closures-tasks.js

```diff
@@ -84,6 +94,8 @@ function getPolynom() {
  *   memoizer() => the same random number  (next run, returns the previous cached result)
  */
 function memoize(func) {
+    let result;
+    return () => !result ? result = func() : result;
```

Comment:
You can just set the result value and then this ternary expression will not be necessary.